src/main/kotlin/no/nav/yrkesskade/meldingmottak/clients/gosys/Oppgave.kt

```diff
@@ -90,4 +90,16 @@ enum class Statuskategori {
     AAPEN, AVSLUTTET
 }
 
+enum class KrutkodeMapping(val brevkode: String?, val behandlingstema: String?, val behandlingstype: String?) {
```

Comment:
Veldig bra! Hadde kanskje vært fint å ha med et beskrivelsefelt også, så er det enklere for oss å holde oversikt over hva som egentlig kommer inn, i fbm feilsøking osv.

Beskrivelser sakset fra kodeverk:

Brev - utland

Ettersendelse til søknad fra selvstendig næringsdrivende og frilansere om opptak i frivillig trygd med rett til særytelser ved yrkesskade

Ettersendelse til melding om yrkesskade eller yrkessykdom som er påført under arbeid på norsk eller utenlandsk landterritorium

Søknad fra selvstendig næringsdrivende og frilansere om opptak i frivillig trygd med rett til særytelser ved yrkesskade